lms/services/moodle.py

```diff
@@ -179,8 +203,8 @@ def _construct_file_tree(course_id, files):
             file_node = {
                 "type": "File",
                 "display_name": path_components[-1],
-                "id": f"moodle://file/url/{file_data['url']}",
-                "lms_id": f"moodle://file/url/{file_data['url']}",
+                "id": f"moodle://file/course/{course_id}/url/{file_data['url']}",
```

Comment:
And oversights for me in the first version of this. We need to store the course alongside the file info to make our lives easier for course copy.